docs/streams/upgrade-guide.html

```diff
@@ -103,21 +103,31 @@
     </ul>
 
     <p>
-        Deprecated methods in <code>KafkaStreams</code>:
+        Modified methods in <code>KafkaStreams</code>:
```

Comment:
Why this? All the method (or some of there overloads were deprecated)

Reply:
The `setStateListener` changes are not deprecated methods.

Reply:
See your point. But `toString()` is deprecated -- should we add two lists? Or say `Deprecated / modified methods ...` ?